tensorhive/models/Group.py

```diff
@@ -61,31 +62,15 @@ def remove_user(self, user: User):
         self.users.remove(user)
         self.save()
 
-    @hybrid_property
-    def as_dict(self):
-        """
-        Serializes current instance into dict.
-        :return: Dictionary representing current instance.
-        """
-        return self._as_dict(True)
-
-    @hybrid_property
-    def as_dict_shallow(self):
+    def as_dict(self, include_users=True):
         """
         Serializes current instance into dict. Will not include group's users (to prevent recurrence).
+        :param include_users: flag that determines if users should be included (False to prevent recurrence)
         :return: Dictionary representing current instance (without users).
         """
-        return self._as_dict(False)
-
-    def _as_dict(self, include_users):
-        group = {
-            'id': self.id,
-            'name': self.name,
-            'createdAt': DateUtils.stringify_datetime(self.created_at),
```

Comment:
`DateUtils.stringify_datetime(self.created_at)` returned date in format `2020-10-29T23:07:30+00:00`. Now date is returned in format `2020-10-29T23:07:30.791980Z`. As a result, different APIs return date in a different formats.

Reply:
fixed in 1a64c0bc9706b0c7d4ca36227f9c6840b84abe03